WordPress/wordpress-playground
# Add onboarding modal to playground

Resolves #497 

## Motivation for the change, related issues

Adds an onboarding modal using the `Modal` from WordPress Components.

## Testing Instructions (or ideally a Blueprint)

1. Checkout the branch
2. Run `npm run dev`
3. Make sure you've cleared any `hideExperimentalNotice` cookie from your Application cache
4. When it initially loads, you should see a modal pop up like the below screenshot.
5. When you click on the blue "ok" button, it should hide and create a cookie to hide the modal.

<img width="840" alt="image" src="https://github.com/user-attachments/assets/e1029528-9712-48d5-9ce3-f289d8e70dec">

## Feedback

@adamziel I worked on this with @sejas , but wanted to check with you specifically on two pieces:

1. Do we want the cookie to be implemented "forever", or to expire in a week or month? Current implementation is forever.
2. Do you have any feedback on the copy? This can definitely say whatever we like, but I'm happy to change it to be whatever you think is appropriate, as well as including or removing emojis 😁.
